tests/system/general/test_35_MaxwellCircuit.py

```diff
@@ -115,3 +115,26 @@ def test_08_import_netlist(self):
         self.aedtapp.insert_design("SchematicImport")
         self.aedtapp.modeler.schematic.limits_mils = 5000
         assert self.aedtapp.create_schematic_from_netlist(self.netlist_file1)
+
+    def test_09_create_voltage_source(self):
+        name_type = ["voltage_source", 123, 1.23]
+        for item in name_type:
+            component = self.aedtapp.modeler.schematic.create_v_sin(name=item, value=3.14, angle=90)
+            assert component.parameters["Name"] == item
+        assert component.parameters["Va"] == 3.14
+
+    def test_10_create_current_source(self):
+        name_type = ["current_source", 456, 4.56]
+        for item in name_type:
+            component = self.aedtapp.modeler.schematic.create_i_sin(name=item, value=2.72, angle=90)
+            assert component.parameters["Name"] == item
+        assert component.parameters["Ia"] == 2.72
+
+    def test_11_create_page(self):
+        assert self.aedtapp.modeler.create_page("string_test")
+        assert self.aedtapp.modeler.nb_pages == 2
+        assert self.aedtapp.modeler.create_page(123)
+        assert self.aedtapp.modeler.nb_pages == 3
+        assert self.aedtapp.modeler.create_page(3.14)
+        assert self.aedtapp.modeler.nb_pages == 4
+        assert not self.aedtapp.modeler.create_page(["create_page_test"])
```

Comment:
@DaveTwyman is there any reason why are you testing create_page giving as argument different type of data types? I think the code lines 134 and 140 are enough to test create_page.

Reply:
@anur7 , The other lines are there to test that passing other data types works correctly. As there is no AEDT documentation on page name data types, we've restricted it to String, Integer and Float and just test these types.